mvx/caip10.md

```diff
@@ -0,0 +1,59 @@
+---
+namespace-identifier: mvx-caip10
+title: MultiversX Namespace - Addresses
+author: MultiversX Team <contact@multiversx.com>
+discussions-to: https://github.com/ChainAgnostic/namespaces/issues/89
+status: Draft
+type: Standard
+created: 2023-09-11
+requires: ["CAIP-2", "CAIP-10"]
+---
+
+# CAIP-10
+
+_For context, see the [CAIP-10][] specification._
+
+## Rationale
+
+The MultiversX Address format is bech32, specified by the [BIP_0173][].
+The human-readable-part of the bech32 addresses is "erd" which means that the MultiversX address always starts with an `erd1` ("1" is a separator) e.g.: `erd1sea63y47u569ns3x5mqjf4vnygn9whkk7p6ry4rfpqyd6rd5addqyd9lf2`.
+
+The MultiversX network defines the address of an account as the bech32 representation of the public key of its corresponding pair of keys (the secret key remains known only to the user that owns the key pair). The public key is 32 bytes in length (64 hex characters), while the address (bech32) is 62 characters long.
+
+On MultiversX, Smart Contracts and only Smart Contracts have their hex (raw) addresses prefixed with 8 zero bytes (meaning that their bech32 address will begin with `erd1qqqqqqqqqqqq` )
+
+## Syntax
+
+```
+caip10-like address:    namespace + ":" chainId + ":" + address
```

Comment:
my only thought is that the `erq` and perhaps also the `1` are redundant, since mvx (and the fact that it's a CAIP-10) tells you it's a MultiversX account and `1` tells you (i presume?) that it's on mainnet.  Maybe you should conserve those 4 redundant characters in case some future form of mvx address is longer and you need the space?

Reply:
Although the first characters may seem redundant for now, it is a lot easier for developers to use the provided bech32 address instead of rebuilding it based on the namespace.

This also remains in sync with other namespaces as the provided address is easy to verify/get the balance of, etc without any other intervention.